README.md

```diff
@@ -1,64 +1,61 @@
+  - [**cdk-sqlite**](./crates/cdk-sqlite/): SQLite Storage backend.
+  - [**cdk-redb**](./crates/cdk-redb/): Redb Storage backend.
+  - [**cdk-rexie**](./crates/cdk-rexie/): Rexie Storage backend for browsers.
+  - [**cdk-axum**](./crates/cdk-axum/): Axum webserver for mint.
+  - [**cdk-cln**](./crates/cdk-cln/): CLN Lightning backend for mint.
+  - [**cdk-lnd**](./crates/cdk-lnd/): Lnd Lightning backend for mint.
+  - [**cdk-strike**](./crates/cdk-strike/): Strike Lightning backend for mint.
+  - [**cdk-lnbits**](./crates/cdk-lnbits/): [LNbits](https://lnbits.com/) Lightning backend for mint.
+  - [**cdk-phoenixd**](./crates/cdk-phoenixd/): Phoenixd Lightning backend for mint.
+  - [**cdk-fake-wallet**](./crates/cdk-fake-wallet/): Fake Lightning backend for mint. To be used only for testing, quotes are automatically filled.
+- Binaries:
+  - [**cdk-cli**](./crates/cdk-cli/): Cashu wallet CLI.
+  - [**cdk-mintd**](./crates/cdk-mintd/): Cashu Mint Binary.
 
 ## Implemented [NUTs](https://github.com/cashubtc/nuts/):
 
 ### Mandatory
 
-| NUT #    | Description                       |
-|----------|-----------------------------------|
-| [00][00] | Cryptography and Models           |
-| [01][01] | Mint public keys                  |
-| [02][02] | Keysets and fees                  |
-| [03][03] | Swapping tokens                   |
-| [04][04] | Minting tokens                    |
-| [05][05] | Melting tokens                    |
-| [06][06] | Mint info                         |
+| NUT #    | Description             |
+| -------- | ----------------------- |
+| [00][00] | Cryptography and Models |
+| [01][01] | Mint public keys        |
+| [02][02] | Keysets and fees        |
+| [03][03] | Swapping tokens         |
+| [04][04] | Minting tokens          |
+| [05][05] | Melting tokens          |
+| [06][06] | Mint info               |
 
 ### Optional
 
-| # | Description | Status
-| --- | --- | --- |
-| [07][07] | Token state check | :heavy_check_mark: |
-| [08][08] | Overpaid Lightning fees | :heavy_check_mark: |
-| [09][09] | Signature restore | :heavy_check_mark: |
-| [10][10] | Spending conditions | :heavy_check_mark: |
-| [11][11] | Pay-To-Pubkey (P2PK) | :heavy_check_mark: |
-| [12][12] | DLEQ proofs | :heavy_check_mark: |
-| [13][13] | Deterministic secrets | :heavy_check_mark: |
+| #        | Description                       | Status             |
+| -------- | --------------------------------- | ------------------ |
+| [07][07] | Token state check                 | :heavy_check_mark: |
+| [08][08] | Overpaid Lightning fees           | :heavy_check_mark: |
+| [09][09] | Signature restore                 | :heavy_check_mark: |
+| [10][10] | Spending conditions               | :heavy_check_mark: |
+| [11][11] | Pay-To-Pubkey (P2PK)              | :heavy_check_mark: |
+| [12][12] | DLEQ proofs                       | :heavy_check_mark: |
+| [13][13] | Deterministic secrets             | :heavy_check_mark: |
```

Comment:
I don't think we need to change this it gets auto formated when rendered so we dont have to add spaces in the raw